From 93f0862a9700393d86c19955313c58f5303f1735 Mon Sep 17 00:00:00 2001
From: Kerong Sui <ksui@codeaurora.org>
Date: Fri, 22 Nov 2013 12:26:41 +0800
Subject: [PATCH 061/176] mm-camera-test: config picture dimension for jpeg
 encoder.

Snapshot dimension is not configured before encoding which fails
the jpeg snapshot function in test app

Change-Id: I15f9a09fbcd94e3713779da5e63dbe06f92049de
---
 QCamera2/stack/mm-camera-test/src/mm_qcamera_snapshot.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/QCamera2/stack/mm-camera-test/src/mm_qcamera_snapshot.c b/QCamera2/stack/mm-camera-test/src/mm_qcamera_snapshot.c
index 669b055..d3c9606 100644
--- a/QCamera2/stack/mm-camera-test/src/mm_qcamera_snapshot.c
+++ b/QCamera2/stack/mm-camera-test/src/mm_qcamera_snapshot.c
@@ -156,6 +156,10 @@ int createEncodingSession(mm_camera_test_obj_t *test_obj,
     encode_param.dest_buf[0].fd = test_obj->jpeg_buf.buf.fd;
     encode_param.dest_buf[0].format = MM_JPEG_FMT_YUV;
 
+    /* main dimension */
+    encode_param.main_dim.src_dim = m_stream->s_config.stream_info->dim;
+    encode_param.main_dim.dst_dim = m_stream->s_config.stream_info->dim;
+
     return test_obj->jpeg_ops.create_session(test_obj->jpeg_hdl,
                                              &encode_param,
                                              &test_obj->current_jpeg_sess_id);
-- 
1.8.3.1

